plasmapy/particles/ionization_state_collection.py

```diff
@@ -39,12 +39,12 @@ class IonizationStateCollection:
         instances with units of number density.
 
     abundances: `dict`, optional, keyword-only
-        A `dict` with `particle_like` elements or isotopes as keys and
+        A `dict` with `ParticleLike` elements or isotopes as keys and
         the corresponding relative abundance as values.  The values must
         be positive real numbers.
 
     log_abundances: `dict`, optional, keyword-only
-        A `dict` with `particle_like` elements or isotopes as keys and
+        A `dict` with `ParticleLike` elements or isotopes as keys and
         the corresponding base 10 logarithms of their relative
         abundances as values.  The values must be real numbers.
```

Comment:
```suggestion
        A `dict` with `~plasmapy.particles.particle_class.ParticleLike`
        objects used as the keys and the corresponding base 10 logarithms of their
        relative abundances as the values.  The values must be real numbers.
```

Fix the `intersphinx` link and re-word a little.  The original wording was really confusing to me so I took a stab clarifying it.  Feel free to do just the `intersphinx` correction if you prefer.